source/src/microapp/cs_MicroappController.cpp

```diff
@@ -231,9 +234,20 @@ MicroappOperatingState MicroappController::getOperatingState(uint8_t appIndex) {
 
 void MicroappController::callMicroapp() {
 	const uint8_t appIndex = 0;
```

Comment:
Obtain this from the actual value `MicroApp::_currentMicroappIndex` with a getter?

Reply:
There's a lot of code that is not ready for multiple microapps yet. Until we actually support that, I don't mind this.